cmd/ore/aws/copy-image.go

```diff
@@ -53,7 +63,49 @@ func runCopyImage(cmd *cobra.Command, args []string) error {
 		os.Exit(1)
 	}
 
-	err = json.NewEncoder(os.Stdout).Encode(amis)
+	out := make(map[string]CopiedImage)
+	for region, ami := range amis {
+		var snapshotID string
+		err = util.RetryConditional(12, 10*time.Second, func(err error) bool {
+			return snapshotID == ""
+		}, func() error {
+			tmpAPI, err := aws.New(&aws.Options{
+				Region:          region,
+				CredentialsFile: credentialsFile,
+				Profile:         profileName,
+			})
+			if err != nil {
+				return fmt.Errorf("creating AWS client in %s region: %v", region, err)
+			}
+			imageName, err := tmpAPI.GetAMIName(ami)
+			if err != nil {
+				return err
+			}
+			// Mantle automatically adds a `-hvm` suffix to HVM images but not to their snapshots
+			// As such we need to trim it from the image name to allow the FindSnapshot function
+			// to find the underlying snapshot of the new AMI
+			imageName = strings.TrimSuffix(imageName, "-hvm")
```

Comment:
@bgilbert I couldn't really decide whether or not I wanted to keep this inside of the `ore` code or directly move it into the `FindSnapshot` API code. Do you have a preference?